### PR TITLE
164766247 Implement ability for phonenumber update

### PR DESCRIPTION
**Pull request by willeswa**
**What does this PR do?**

Allow user to update their phone number on the system

**Description of Task to be completed?**

- [x] Write tests for the specified feature
- [x] Add the code for the specified feature

**How should this be manually tested?**

After cloning the repository to your local machine:
* Use Postman to add an account with your personal information
* Signin using the information
* Send a PATCH request to the '/users/2/phonenumber' with a new number
* Retrieve your profile information (alternatively, you can check the db records)
* You should be able to see that your number has been update to the new number

**What are the relevant pivotal tracker stories?**

#164766247

**Relevant Screenshots**

Screenshot from 2019-03-21 14-51-20
Screenshot from 2019-03-21 14-54-00
Screenshot from 2019-03-21 14-54-27

### CODE REVIEW

**Comment by wangonya** [comment]
This looks ok to me. Maybe the rest of the team can give more input.

**Comment by philophilo** [comment]
The PR has commit messages at the very, clear them out. It is distorting the PR format. Kindly also edit the commit message to appear in this format
```
164766247-story(user model): implement ability for phonenumber update

- allow user to change phone number on necessity
- user data such as contact information is bound to changes
- there is need to allow for update of the same in the system

[Finishes #164766247]
```

File: app/api/v2/views/users.py [hunk 1]
```diff
@@ -143,6 +143,8 @@ def reset_password():
     }])
 
 # send the email securely from server
+
+
```
Comment (philophilo) [inline app/api/v2/views/users.py]: Lets keep unnecessary lines out

**Comment by wekesa931** [comment]
It is a good feature and the commit message is clear

File: app/api/v2/views/users.py (hunk 1 above)
Comment (willeswa) [reply app/api/v2/views/users.py]: I'll look at it

**Pull request closed without merging**